.gitignore

```diff
@@ -0,0 +1,115 @@
+# Created by https://www.gitignore.io/api/python
+# Edit at https://www.gitignore.io/?templates=python
+
+### Python ###
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
+pip-wheel-metadata/
+share/python-wheels/
+*.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
+
+# PyInstaller
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
+
+# Installer logs
+pip-log.txt
+pip-delete-this-directory.txt
+
+# Unit test / coverage reports
+htmlcov/
+.tox/
+.nox/
+.coverage
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+*.cover
+.hypothesis/
+.pytest_cache/
+
+# Translations
+*.mo
+*.pot
+
+# Scrapy stuff:
+.scrapy
+
+# Sphinx documentation
```

Comment:
The way the doc currently works is that I build the source in `/docs/build`, commit and upload. Github pages does the rest. So maybe this can go too?

Reply:
Built artifacts probably are best left to a CI system to build, and then pushed to a `gh-pages` branch; this is a common pattern in many open source projects. I remember when merging the latest master into my fork, I had a ton of conflicts to resolve manually for whatever reason. If you still prefer, I can remove it, but I think encouraging common community practice is better.

Reply:
I agree with you, the way it is currently done is very hacky and this is something CI should take care of. If you can do it quickly it would be amazing, otherwise we can do it in another PR and keep this line for now.

Reply:
Let's do it in another PR.

Do you like Sphinx? I've been test-driving mkdocs recently, and with the right plugins, it's possible to make super cool sites like the following ones:

* http://bayesian-analysis-recipes.ericmjl.com/
* https://fastapi.tiangolo.com/

API docs are possible too, with auto-parsing of docstrings. My experiments with the PyMC4 docs tell me it's doable, and they look gorgeous. But the tradeoff is docstrings have to be written in Markdown, so I wouldn't recommend it for a library that is extensively invested in sphinx-style docstrings. Let me know; I'll keep in mkdocs for now unless you choose otherwise.

Reply:
I don’t think anyone is a fan of Sphinx, but it does the job quite well. In particular the API docs which I find very important. I don’t know mkdocs very well and I am not downright opposed to it. Let’s see how this works for PyMC4 first :)